zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/incident/JobWorkerElementIncidentTest.java

```diff
@@ -138,6 +138,36 @@ public void shouldCreateIncidentIfJobTypeExpressionOfInvalidType() {
         .hasVariableScopeKey(recordThatLeadsToIncident.getKey());
   }
 
+  @Test
+  public void shouldCreateIncidentIfJobTypeExpressionIsEmptyString() {
+    // given
+    ENGINE.deployment().withXmlResource(process(t -> t.zeebeJobTypeExpression("\"\""))).deploy();
+
+    // when
+    final long processInstanceKey = ENGINE.processInstance().ofBpmnProcessId(PROCESS_ID).create();
+
+    final var recordThatLeadsToIncident =
+        RecordingExporter.processInstanceRecords(ProcessInstanceIntent.ELEMENT_ACTIVATING)
+            .withProcessInstanceKey(processInstanceKey)
+            .withElementType(elementBuilder.getElementType())
+            .getFirst();
+
+    // then
+    final var incidentCreated =
+        RecordingExporter.incidentRecords(IncidentIntent.CREATED)
+            .withProcessInstanceKey(processInstanceKey)
+            .getFirst();
+
+    Assertions.assertThat(incidentCreated.getValue())
+        .hasErrorType(ErrorType.EXTRACT_VALUE_ERROR)
+        .hasErrorMessage("Expected result of the expression '\"\"' to be not empty, but was ''.")
```

Comment:
🔧  Interesting that the result is displayed here as `''` rather than `'""'`. That's correct, but we could clarify it slightly:
>Expected result of the expression '""' to be a not-empty string, but was an empty string ''.

Likewise, a variable `x` with value empty string `""` would result in the following message:
>Expected result of the expression 'x' to be a not-empty string, but was an empty string ''.

@ana-vinogradova-camunda What do you think?

Reply:
Yes, clarifying it this way would be better. I will make the change 👍